### DISCUSSION TITLE
Incomplete Data: LotTender Present in LotResult but Absent in Associated SettledContract

### DISCUSSION THREAD

**Discussion by sebastian-hermix** [OP-TED/eForms-SDK#895] [Q&A] [answered]
Link to notice: https://ted.europa.eu/en/notice/-/detail/202368-2024#section6.1_29
In the first LotResult in xml we have TEN-0001 declared as winner on Lot-0001 but in the SettledContract only TEN-0003 is present.
```
<efac:LotResult>
   <cbc:ID>RES-0001</cbc:ID>
   <efac:LotTender>
      <cbc:ID>TEN-0003</cbc:ID>
   </efac:LotTender>
   <efac:SettledContract>
      <cbc:ID>CON-0001</cbc:ID>
   </efac:SettledContract>
   <efac:TenderLot>
      <cbc:ID>LOT-0001</cbc:ID>
   </efac:TenderLot>
</efac:LotResult>
```
```
<efac:SettledContract>
   <cbc:ID>CON-0001</cbc:ID>
   <cbc:IssueDate>2022-02-25+01:00</cbc:IssueDate>
   <cbc:Title languageID="POL">Zadanie nr 1 – Stare Miasto</cbc:Title>
   <efac:ContractReference>
      <cbc:ID>1</cbc:ID>
   </efac:ContractReference>
   <efac:LotTender>
      <cbc:ID>TEN-0001</cbc:ID>
   </efac:LotTender>
</efac:SettledContract>
```
LotTenders within a SettledContract should also include LotTenders from LotResult? because I was expecting to find TEN-0003 in the LotTenders of CON-0001.
In the interest of technical clarity, could you please elucidate this issue?

**Comment by YvesJo** [comment, marked as answer]
Hi,
Data is complete but inconsistent; an additional rule may be necessary to prevent such errors.
A LotTender (TEN-XXXX) for a given Lot may only appear in LotResults for that Lot.
It is not necessary to have all the won lots in a single contract, each could have its own contract.
LotTender TEN-0003 has been defined for Lot 3 (LOT-0003) but appears in LotResults for Lot 1 (RES-0001) and Lot 3 (RES-0003).
The ground cause of the inconsistent information is that the LotTender associated to LotResult for Lot 1 (RES-0001) should be TEN-0001 instead of TEN-0003.
